tests/components/bluetooth/__init__.py

```diff
@@ -27,8 +28,27 @@
     "inject_bluetooth_service_info",
     "patch_all_discovered_devices",
     "patch_discovered_devices",
+    "generate_advertisement_data",
 )
 
+ADVERTISEMENT_DATA_DEFAULTS = {
+    "local_name": "",
+    "manufacturer_data": {},
+    "service_data": {},
+    "service_uuids": [],
+    "rssi": -127,
+    "platform_data": ((),),
+    "tx_power": -127,
+}
```

Comment:
This is rather annoying. Will keep on breaking in updates. This need to move into bleak.

Reply:
I mean untill its fully removed in next bump

Reply:
`bleak` broke it on purpose to require the args to be set to prevent construction errors so I think we are stuck with this.